src/Database/Schema/Table.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace TCG\Voyager\Database\Schema;
+
+use Doctrine\DBAL\Schema\Table as DoctrineTable;
+use Doctrine\DBAL\Schema\Comparator;
+
+class Table extends DoctrineTable
+{
+    public static function make($table)
+    {
+        if (!is_array($table)) {
+            $table = json_decode($table, true);
+        }
+
+        $name = Identifier::validate($table['name'], 'Table');
+       
+        $columns = [];
+        foreach ($table['columns'] as $columnArr) {
+            $column = Column::make($columnArr);
+            $columns[$column->getName()] = $column;
+        }
```

Comment:
Refactor idea.

```
$columns = collect($table['columns'])->transform(function ($column) {
    return Column::make($column);
})->all();
```

However one problem... The key is not created now, there must be a method for that.